lib/kafo/help_builders/base.rb

```diff
@@ -82,7 +82,11 @@ def params_mapping(items)
       end
 
       def parametrization
-        @parametrization ||= Hash[@params.map { |p| [parametrize(p), p] }]
+        @parametrization ||= begin
+          @params.inject({}) do |h,p|
+            h.update(parametrize(p) => p, parametrize(p, 'reset-') => p)
```

Comment:
not tested yet, but is this necessary for each parameter? maybe we could document it somewhere just once to improve readability